.travis.yml

```diff
@@ -78,8 +78,8 @@ jobs:
       api_key: "$GITHUB_TOKEN1"
       file_glob: true
       file:
-      - bin/Debug/Lachain.CommunicationHub.Net.0.0.*.nupkg
-      - ../Lachain.CommunicationHub.Native.0.0.*.nupkg
+      - Lachain.CommunicationHub.Native.*nupkg
```

Comment:
it should be `../Lachain.CommunicationHub.Native.*nupkg`

Reply:
fixed